packages/http-client-csharp/generator/Microsoft.Generator.CSharp.ClientModel/src/Providers/MrwSerializationTypeDefinition.cs

```diff
@@ -965,7 +1068,25 @@ private List<ParameterProvider> BuildSerializationConstructorParameters()
                     _rawDataField.Type));
             }
 
-            return constructorParameters;
+            return (constructorParameters, constructorInitializer);
+
+            static ConstructorSignature? GetBaseConstructor(MrwSerializationTypeDefinition? baseProvider)
```

Comment:
nit: missing access modifier

Reply:
local methods cannot have access modifiers.
I defined this method as a local method because only the enclosing method needs it. Putting it inside the class might pollute the IntelliSense.